src/main/java/co/inventorsoft/mailsecurity/models/User.java

```diff
@@ -0,0 +1,34 @@
+package co.inventorsoft.mailsecurity.models;
+
+import co.inventorsoft.mailsecurity.forms.UserForm;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
```

Comment:
When you work with JDBC, you do not need Entity.
Entity is part of ORM (javax).